### ISSUE TITLE
[Jakarta EE TCK Drive folder]  Public Access Creation

### ISSUE THREAD

**Issue by aeiras** [jakartaee/collateral#3]
Hola @ivargrimstad, 

Can you please create a folder with public access in the Jakarta EE Community drive called Jakarta EE TCK?

After today's TCK call, we need the space to add the minutes from the public call. :) Thank you!

**Comment by keilw** [comment]
@aeiras @ivargrimstad It's nice to help creating more TCK tests, but is there a plan when TomEE might pass the existing TCK like all the other vendors that did it, some within a week after Jakarta EE 8 went Final? ;-)

**Comment by ivargrimstad** [comment]
@aeiras I'm not the one responsible for maintaining this drive. It is handled by the marketing committee. As far as I know, they have appointed a group of maintainers.

**Comment by keilw** [comment]
So is this only for Marketing purposes? If not, how should it properly correlate with the actual Jakarta EE TCK repo?

**Comment by ivargrimstad** [comment]
> So is this only for Marketing purposes? If not, how should it properly correlate with the actual Jakarta EE TCK repo?

I don't think it is marketing. The Google Drive is maintained by the marketing committee.

**Comment by keilw** [comment]
Still keen to see, how this may work with the actual TCK e.g. automated tests and CI jobs like Jenkins? We had quite some bad experience with eager contributors writing JUnit tests or even TCK tests for JavaMoney and a lot of it had to be reverted because they did not understand the JSR enough, so I fear the test quality might be equally bad here unless it is also tested (automatically) in some form in the actual TCK environment.
